dev/proto-generate.sh

```diff
@@ -10,9 +10,9 @@ yarn --cwd ./bindings/typescript install --frozen-lockfile
 echo "--- cargo install rust-protobuf ---"
 PROTOC_GEN_RUST_VERSION="$(cat bindings/rust/Cargo.toml | grep 'protobuf =' | sed -E 's/.*\"=(.+)\"/\1/g')"
 if ! grep -q "$PROTOC_GEN_RUST_VERSION" "./.bin/PROTOC_GEN_RUST_VERSION" \
-   || ! test -f "./.bin/bin/protoc-gen-rust"; then
+   || ! test -f "./.bin/bin/protoc-gen-rs"; then
```

Comment:
This update is because of the binary name change: https://github.com/stepancheg/rust-protobuf/blob/master/CHANGELOG.md